exec/vector/src/main/java/org/apache/drill/exec/vector/accessor/convert/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Defines the type conversion mechanism along with several basic
+ * conversions.
+ */
+package org.apache.drill.exec.vector.accessor.convert;
```

Comment:
Please add conversion for decimal and float.

Reply:
Now is "later"... Added those which are type-save (widening conversions.) How do we want to handle overflow in narrowing conversions? SQL says overflow is an error, so do we want to fail the query? Set the value to null? Truncate the high-order bits?

For float-to-integral conversions, SQL says that the behavior is implementation-specific. Do we want to round using Java semantics?